app/models/message.rb

```diff
@@ -0,0 +1,19 @@
+class Message < ApplicationRecord
+  include SymbolizeJSON
+
+  has_many :matched_messages
+  has_many :subscription_contents
+
+  validates_presence_of :title, :body
+  validates :url, absolute_path: true, allow_nil: true
```

Comment:
I wonder if we need to rename this absolute_path validator too? From a very non-authoritive looking website I've found the name seems to be root-relative, so we could maybe call the validator RootRelativeUrlValidator?

Reply:
I like `RootRelativeUrlValidator`... I'll do that 👍